README.md

```diff
@@ -0,0 +1,27 @@
+# Cellular-Connected Electronic Kiosk
+
+A cellular-based solution for downloading resources for an electronic kiosk display without an Internet connected, using a simple Python script.
```

Comment:
connected -> connection

Reply:
I see you found my copy pasta. Good catch (again) 😬